// design-tokens-local/build/tokens.js
/**
 * Do not edit directly
 * Generated on Mon, 31 Oct 2022 13:02:36 GMT
 */

export const TokenSetOrder0 = "global";
export const Space100 = "8px";
export const Space150 = "12px";
export const Space200 = "16px";
export const Space250 = "20px";
export const Space300 = "24px";
export const Space400 = "32px";
export const Space500 = "40px";
export const Space600 = "48px";
export const Space700 = "56px";
export const Space800 = "64px";
export const Space900 = "72px";
export const Space1000 = "80px";
export const Space1100 = "88px";
export const Space1200 = "96px";
export const Space1500 = "120px";
export const Space1700 = "136px";
export const Space2000 = "160px";
export const Space2300 = "184px";
export const Space2500 = "200px";
export const Space3000 = "240px";
export const Space4000 = "320px";
export const Space5000 = "400px";
export const Space050 = "4px";
export const Radius0 = "0px";
export const Radius100 = "4px";
export const Radius200 = "8px";
export const Radius300 = "12px";
export const Radius400 = "16px";
export const Radius500 = "24px";
export const Border0 = "0px";
export const Border100 = "1px";
export const Border200 = "1.5px";
export const Border300 = "2px";
export const Border400 = "3px";
export const Border500 = "4px";
export const Elevation0 = {"x":0,"y":0,"blur":0,"spread":0,"color":"#000000","type":"dropShadow"};
export const Elevation100 = [{"color":"#24243514","type":"dropShadow","x":0,"y":4,"blur":4,"spread":-2},{"color":"#2424351f","type":"dropShadow","x":0,"y":2,"blur":4,"spread":-2}];
export const Elevation200 = [{"color":"#24243517","type":"dropShadow","x":0,"y":8,"blur":8,"spread":-4},{"color":"#2424351c","type":"dropShadow","x":0,"y":4,"blur":6,"spread":-4}];
export const Elevation300 = [{"color":"#2424351a","type":"dropShadow","x":0,"y":8,"blur":24,"spread":-4},{"color":"#2424351a","type":"dropShadow","x":0,"y":6,"blur":12,"spread":-6}];
export const Elevation400 = [{"color":"#2424351f","type":"dropShadow","x":0,"y":12,"blur":42,"spread":-4},{"color":"#2424351f","type":"dropShadow","x":0,"y":8,"blur":18,"spread":-6}];
export const Elevation500 = [{"color":"#2424351a","type":"dropShadow","x":0,"y":18,"blur":88,"spread":-4},{"color":"#2424351a","type":"dropShadow","x":0,"y":12,"blur":28,"spread":-6}];
export const BaseColorsGreen100 = "#EFFDE3";
export const BaseColorsGreen200 = "#DDFBC9";
export const BaseColorsGreen300 = "#C2F4AB";
export const BaseColorsGreen400 = "#A8EA92";
export const BaseColorsGreen500 = "#81DD6E";
export const BaseColorsGreen600 = "#5BBE51";
export const BaseColorsGreen700 = "#39A037";
export const BaseColorsGreen800 = "#237F28";
export const BaseColorsGreen900 = "#166A1F";
export const BaseColorsIndigo100 = "#D9D8FD";
export const BaseColorsIndigo200 = "#B3B2FB";
export const BaseColorsIndigo300 = "#8B89F3";
export const BaseColorsIndigo400 = "#6C69E8";
export const BaseColorsIndigo500 = "#3E3BD9";
export const BaseColorsIndigo600 = "#2D2BBA";
export const BaseColorsIndigo700 = "#1F1D9C";
export const BaseColorsIndigo800 = "#14127D";
export const BaseColorsIndigo900 = "#0C0B68";
export const BaseColorsYellow100 = "#FFFAE1";
export const BaseColorsYellow200 = "#FFEFB3";
export const BaseColorsYellow300 = "#FFE48D";
export const BaseColorsYellow400 = "#FFD871";
export const BaseColorsYellow500 = "#FFC641";
export const BaseColorsYellow600 = "#DBA22F";
export const BaseColorsYellow700 = "#B78121";
export const BaseColorsYellow800 = "#926115";
export const BaseColorsYellow900 = "#7B4B0D";
export const BaseColorsRed100 = "#FEDDD8";
export const BaseColorsRed200 = "#FFBBB2";
export const BaseColorsRed300 = "#FD9789";
export const BaseColorsRed400 = "#FC736A";
export const BaseColorsRed500 = "#ED4646";
export const BaseColorsRed600 = "#D62F29";
export const BaseColorsRed700 = "#B31C23";
export const BaseColorsRed800 = "#901122";
export const BaseColorsRed900 = "#770A21";
export const BaseColorsBlue100 = "#E7EEFD";
export const BaseColorsBlue200 = "#D6E2FC";
export const BaseColorsBlue300 = "#A2BEE8";
export const BaseColorsBlue400 = "#80A6DF";
export const BaseColorsBlue500 = "#5A8CD6";
export const BaseColorsBlue600 = "#2365C8";
export const BaseColorsBlue700 = "#205CB6";
export const BaseColorsBlue800 = "#194990";
export const BaseColorsBlue900 = "#143A72";
export const BaseColorsGrey100 = "#F8F9FC";
export const BaseColorsGrey200 = "#F1F3F9";
export const BaseColorsGrey300 = "#DEE3ED";
export const BaseColorsGrey400 = "#C2C9D6";
export const BaseColorsGrey500 = "#8F96A3";
export const BaseColorsGrey600 = "#5E636E";
export const BaseColorsGrey700 = "#2F3237";
export const BaseColorsGrey800 = "#1D1E20";
export const BaseColorsGrey900 = "#111213";
export const BaseColorsBrandNight = "#242435";
export const BaseColorsBrandCoral = "#FF5A65";
export const BaseColorsBrandSoftBlue = "#D6E2FC";
export const BaseColorsBrandSoftPink = "#F2B5E8";
export const BaseColorsBrandEnergy = "#EFF258";
export const BaseColorsBrandWhite = "#ffffff";
export const BaseColorsBrandBlack = "#000000";
export const BaseColorsCoral100 = "#FFF8F8";
export const BaseColorsCoral200 = "#FFE9E7";
export const BaseColorsCoral300 = "#FFA3A2";
export const BaseColorsCoral400 = "#FF7C7E";
export const BaseColorsCoral500 = "#FF5A65";
export const BaseColorsCoral600 = "#C3484F";
export const BaseColorsCoral700 = "#A13E42";
export const BaseColorsCoral800 = "#7E3335";
export const BaseColorsCoral900 = "#62292B";
export const BaseColorsNight100 = "#EDEDEF";
export const BaseColorsNight200 = "#D4D4D8";
export const BaseColorsNight300 = "#BCBBC2";
export const BaseColorsNight400 = "#A4A3AC";
export const BaseColorsNight500 = "#8B8A94";
export const BaseColorsNight600 = "#72717D";
export const BaseColorsNight700 = "#5E5D6B";
export const BaseColorsNight800 = "#4A4959";
export const BaseColorsNight900 = "#242435";
export const BaseColorsPink100 = "#FCE8F8";
export const BaseColorsPink200 = "#F6C6ED";
export const BaseColorsPink300 = "#F2B5E8";
export const BaseColorsPink400 = "#E3AADA";
export const BaseColorsPink500 = "#C695BE";
export const BaseColorsPink600 = "#A77EA0";
export const BaseColorsPink700 = "#896883";
export const BaseColorsPink800 = "#71576D";
export const BaseColorsPink900 = "#594555";
export const BaseColorsEnergy100 = "#FCFDC4";
export const BaseColorsEnergy200 = "#EFF258";
export const BaseColorsEnergy300 = "#D9DB51";
export const BaseColorsEnergy400 = "#C0C249";
export const BaseColorsEnergy500 = "#A8A942";
export const BaseColorsEnergy600 = "#8E8F3A";
export const BaseColorsEnergy700 = "#757531";
export const BaseColorsEnergy800 = "#61612B";
export const BaseColorsEnergy900 = "#4D4D24";
export const Medium = "Medium";
export const Inter = "Inter";
export const ComponentButtonMedium = {"fontFamily":"Inter","fontWeight":"Medium","lineHeight":14,"fontSize":14};
export const ComponentButtonLarge = {"fontFamily":"Inter","fontWeight":"Medium","lineHeight":16,"fontSize":16};
export const ColorBackgroundActionStatic = "#3e3bd9";
export const ColorBackgroundActionHover = "#2d2bba";
export const ColorBackgroundActionFocus = "#2d2bba";
export const ColorBackgroundActionActive = "#2d2bba";
export const ColorBackgroundDisabled = "#ededef";
export const ColorBackgroundDangerStatic = "#d62f29";
export const ColorBackgroundDangerHover = "#b31c23";
export const ColorBackgroundDangerFocus = "#b31c23";
export const ColorBackgroundDangerActive = "#b31c23";
export const ColorBackgroundBody = "#ffffff";
export const ColorTextOnAction = "#ffffff";
export const ColorTextOnDisabled = "#72717d";
export const ColorTextActionStatic = "#3e3bd9";
export const ColorTextActionHover = "#2d2bba";
export const ColorTextActionFocus = "#3e3bd9";
export const ColorTextActionActive = "#eff258";
export const ColorTextActionDisabled = "#72717d";
